src/mbgl/gfx/fill_extrusion_drawable_data.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
Is this drawable data being used or do you have plans for it to be used?

Reply:
No, it turned out to not be needed, and I forgot about it.